src/__mocks__/api-handlers.js

```diff
@@ -235,7 +235,7 @@ function githubHandlers(githubStore) {
     rest.delete(`${GH_BASE}/repos/:org/:repo/issues/comments/:commentId`, (req, res, ctx) => {
       const {org, repo, commentId} = req.params
 
-      if (org !== 'bldrs-ai' || repo !== 'Share' || !commentId) {
+      if (org !== 'pablo-mayrgundter' || repo !== 'Share' || !commentId) {
```

Comment:
Let's move these to one of the mock test IDs to make it clear that it's not prod data, and also to free my ghost from the machine :)